draft-ietf-quic-transport.md

```diff
@@ -2830,13 +2830,15 @@ valid frames? -->
 
 ### Sending ACK Frames
 
-<!-- TODO: Re-read this section for flow and redundancy. -->
-
-To avoid creating an indefinite feedback loop, an endpoint MUST NOT send an ACK
-frame in response to a packet containing only ACK or PADDING frames, even if
-there are packet gaps which precede the received packet.  The endpoint MUST
-however acknowledge packets containing only ACK or PADDING frames when sending
-ACK frames in response to other packets.
+ACK frames are sent to acknowledge received packets and indicate the frames
+within them has been processed. An endpoint MUST NOT send more than one packet
```

Comment:
The transition to the second sentence is a little jarring.  I think that you could make it a new paragraph.

Reply:
After re-reading the text in the section before, I decided to drop this new sentence.